pkg/ccl/importccl/read_import_mysql.go

```diff
@@ -183,25 +183,30 @@ func mysqlValueToDatum(
 		}
 
 	case *mysql.UnaryExpr:
-		if v.Operator != "-" {
-			return nil, errors.Errorf("unexpected operator: %q", v.Operator)
-		}
-		parsed, err := mysqlValueToDatum(v.Expr, desired, evalContext)
-		if err != nil {
-			return nil, err
-		}
-		switch i := parsed.(type) {
-		case *tree.DInt:
-			return tree.NewDInt(-*i), nil
-		case *tree.DFloat:
-			return tree.NewDFloat(-*i), nil
-		case *tree.DDecimal:
-			dec := &i.Decimal
-			dd := &tree.DDecimal{}
-			dd.Decimal.Neg(dec)
-			return dd, nil
+		switch v.Operator {
+		case "-":
+			parsed, err := mysqlValueToDatum(v.Expr, desired, evalContext)
+			if err != nil {
+				return nil, err
+			}
+			switch i := parsed.(type) {
+			case *tree.DInt:
+				return tree.NewDInt(-*i), nil
+			case *tree.DFloat:
+				return tree.NewDFloat(-*i), nil
+			case *tree.DDecimal:
+				dec := &i.Decimal
+				dd := &tree.DDecimal{}
+				dd.Decimal.Neg(dec)
+				return dd, nil
+			default:
+				return nil, errors.Errorf("unsupported negation of %T", i)
+			}
+		case "_binary", "_binary ":
+			// TODO(dt): do something with this?
```

Comment:
this todo is kind of vague. can it have a bit more description?

Reply:
Done